src/targeting.js

```diff
@@ -609,6 +610,27 @@ export function newTargeting(auctionManager) {
     });
   }
 
+  function getAdUnitTargeting(adUnitCodes) {
```

Comment:
perhaps a comment here would be helpful, example of how `ADSERVER_TARGETING` is getting set.